src/geo/ui/search.js

```diff
@@ -245,7 +245,7 @@ var Search = View.extend({
   clean: function() {
     this._unbindEvents();
     this._destroySearchPin();
-    this.elder('clean');
+    Search.__super__.clean.call(this);
```

Comment:
What about `View.prototype.clean.call(this)`? We're using this pattern already and `__super__` feels a bit hacky to me.